<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>ev override</string>
       </attribute>
       <widget class="PhotoGallery" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>0</y>
          <width>600</width>
          <height>600</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>ev central</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="LabelCentralEvent" name="label_2">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>ev filter</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="LabelWithFilterEvent" name="label_3">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>custom event and redirect event</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_source">
          <property name="text">
           <string>Source</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomDestinationLabel" name="label_destination">
          <property name="text">
           <string>Destination</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>hw - mywidget</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAppFilter"/>
    <addaction name="actionCustomAppFilter"/>
    <addaction name="actionLabelFilter"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAppFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>AppFilter</string>
   </property>
  </action>
  <action name="actionCustomAppFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CustomAppFilter</string>
   </property>
  </action>
  <action name="actionLabelFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LabelFilter</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhotoGallery</class>
   <extends>QLabel</extends>
   <header>photogallery.h</header>
  </customwidget>
  <customwidget>
   <class>LabelCentralEvent</class>
   <extends>QLabel</extends>
   <header>labelcentralevent.h</header>
  </customwidget>
  <customwidget>
   <class>LabelWithFilterEvent</class>
   <extends>QLabel</extends>
   <header>labelwithfilterevent.h</header>
  </customwidget>
  <customwidget>
   <class>CustomDestinationLabel</class>
   <extends>QLabel</extends>
   <header>customdestinationlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
